cpg_utils/config.py

```diff
@@ -112,8 +110,7 @@ def read_configs(config_paths: List[str]) -> frozendict:
     """
 
     config: dict = {}
-    template_path = os.path.join(os.path.dirname(__file__), 'config-template.toml')
-    for path in [template_path] + config_paths:
+    for path in config_paths:
```

Comment:
As discussed in https://centrepopgen.slack.com/archives/C018KFBCR1C/p1666918967093519, we probably want to retain the documentation purpose of `config-template.toml` somehow, but that might belong in a different repository.

Reply:
@lgruen see https://github.com/populationgenomics/team-docs/pull/126 assigned to you